app.js

```diff
@@ -23,4 +23,27 @@ function Player (loginName, password, playerName) {
 new Player('Jonny', '1234', 'Mark');
 
 // check();
+
 //add create an account
+
+//life source = incrementing and decrementing oxygen/health level
+
+var lifeSource = 0 //decide on a number to start with.
+
+function drawScore(score) {
```

Comment:
@abdih17 what is this intended to do? I don't think we need to be drawing a chart.